lib/handlers/authorize-handler.js

```diff
@@ -28,7 +28,7 @@ var url = require('url');
 
 var responseTypes = {
   code: require('../response-types/code-response-type'),
-  token: require('../response-types/token-response-type')
+  //token: require('../response-types/token-response-type')
```

Comment:
The `token` property will be used once PR #271 is merged in. The check using `_.has` in `getResponseType` is actually cleaner IMO and mirrors what's done for `grant_type` in [TokenHandler#handleGrantType()](https://github.com/oauthjs/node-oauth2-server/blob/master/lib/handlers/token-handler.js#L211).